src/domain/dtos/shlink.ts

```diff
@@ -0,0 +1,13 @@
+
+export class CreateShlinkDto {
+    userId: string
+    passcodeFailuresRemaining: number
+    configPasscode?: string
+    configExp?: Date
+    active: boolean
+    managementToken: string
```

Comment:
Please make this db generated

Reply:
db generated for active and managementToken right?